OJP/OJP_PlaceSupport.xsd

```diff
@@ -267,6 +272,21 @@
 			</xs:element>
 		</xs:choice>
 	</xs:complexType>
+	<xs:complexType name="PointOfInterestAdditionalInformationStructure">
+		<xs:annotation>
+			<xs:documentation>Generic structure for additional information on POIs.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Category" type="xs:string" minOccurs="0"/>
+			<xs:element name="Attributes" type="KeyValueType" maxOccurs="unbounded"/>
```

Comment:
Already unbounded inside the PointOfInterest->AdditionalInformation ... do we really need 2 unbounded levels

Reply:
rewritenn